.github/workflows/nightly.yml

```diff
@@ -0,0 +1,59 @@
+name: Nightly
+
+on:
+  schedule:
+    - cron: '0 1 * * *'   # everyday at 02:00 UTC+1
+  workflow_dispatch:
+
+jobs:
+  build-macos:
+    name: build-macos
+    runs-on: macos-12
+    outputs:
+      build_result: ${{ steps.build.outcome }}
+    strategy:
+      matrix:
+        target: ['ia32-generic-qemu']
+        include:
+          - target: 'ia32-generic-qemu'
+            syspage: 'psh pc-ata uart16550'
+    steps:
+      - name: Install denependecies
+        run: |
+          brew install bash coreutils autoconf automake genext2fs make wget gnu-sed
+          echo "$(brew --prefix make)/libexec/gnubin" >> $GITHUB_PATH
+          echo "$(brew --prefix gnu-sed)/libexec/gnubin" >> $GITHUB_PATH
+
+      - name: Download toolchain
+        uses: dawidd6/action-download-artifact@v2
+        with:
+          workflow: build-toolchain.yml
+          path: /Users/runner
```

Comment:
shouldn't this be some GITHUB env variable? Is this value future-proof?

Reply:
Now I use workspace dir